src/mlpack/tests/activation_functions_test.cpp

```diff
@@ -751,6 +751,54 @@ TEST_CASE("ReLU6FunctionTest", "[ActivationFunctionsTest]")
   CheckReLU6Correct(activationData, desiredActivations, desiredDerivatives);
 }
 
+/**
+ * Implementation of the Threshold activation function derivative test. The function
+ * is implemented as Threshold layer in the file threshold_fn.hpp.
+ *
+ * @param input Input data used for evaluating the Threwshold activation function.
+ * @param target Target data used to evaluate the Threshold activation.
+ */
+void CheckThresholdCorrect(const arma::colvec input,
+                       const arma::colvec ActivationTarget,
+                       const arma::colvec DerivativeTarget)
+{
```

Comment:
Would you fix the indentation her please?

Reply:
Done